hoomd/mpcd/test-py/init_make_random_sphere_test.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2009-2019 The Regents of the University of Michigan
+# This file is part of the HOOMD-blue project, released under the BSD 3-Clause License.
+
+# Maintainer: mphoward
+
+import unittest
+import hoomd
+import numpy as np
+from hoomd import mpcd
+
+# unit tests for generating random solvent particles inside sphere
+class mpcd_snapshot(unittest.TestCase):
+    def setUp(self):
+        hoomd.context.initialize()
+        hoomd.init.read_snapshot(hoomd.data.make_snapshot(N=0, box=hoomd.data.boxdim(L=120.)))
+
+    def test_init(self):
```

Comment:
Should we add a `test_init_2d` that makes sure this works for 2d initialization, since the code is technically supposed to work for this? It probably would not be too hard to copy paste this test, and change it to use polar coordinates